src/FsAutoComplete.Core/CommandResponse.fs

```diff
@@ -285,7 +285,41 @@ module CommandResponse =
     Position : Pos
   }
 
-
+  type WorkspacePeekResponse = {
+    Found: WorkspacePeekFound list
+  }
+  and WorkspacePeekFound =
+    | Directory of WorkspacePeekFoundDirectory
+    | Solution of WorkspacePeekFoundSolution
+  and WorkspacePeekFoundDirectory = {
+    Directory: string
+    Fsprojs: string list
+  }
+  and WorkspacePeekFoundSolution = {
+    Path: string
+    Items: WorkspacePeekFoundSolutionItem list
+    Configurations: WorkspacePeekFoundSolutionConfiguration list
+  }
+  and [<RequireQualifiedAccess>] WorkspacePeekFoundSolutionItem = {
+    Guid: Guid
+    Name: string
+    Kind: WorkspacePeekFoundSolutionItemKind
+  }
+  and WorkspacePeekFoundSolutionItemKind =
+    | MsbuildFormat of WorkspacePeekFoundSolutionItemKindMsbuildFormat
+    | Folder of WorkspacePeekFoundSolutionItemKindFolder
+  and [<RequireQualifiedAccess>] WorkspacePeekFoundSolutionItemKindMsbuildFormat = {
+    Configurations: WorkspacePeekFoundSolutionConfiguration list
+  }
+  and [<RequireQualifiedAccess>] WorkspacePeekFoundSolutionItemKindFolder = {
+    Items: WorkspacePeekFoundSolutionItem list
+    Files: string list
```

Comment:
The files inside a solution folder, in the list the full paths.
Is in another property instead of WorkspacePeekFoundSolutionItem, because a file can be only inside a solution folder, not at root level (solution) like a project